### PR TITLE
Checking whether this pull request makes sense

### PR DESCRIPTION
**Pull request by lsawade**
Hi Peter,

I think I didn't pull something from your branch at some point, and now it's fast-forwarding and rewinding and stuff. There were 0 merge conflicts, but before merging this I/we should go through at least some of the changes, just to see whether they make sense. 

- [x] Tests are passing
- [x] No merge-conflicts
- [ ] Am I breaking something?

### CODE REVIEW

**Comment by PeterMakus** [comment]
Hi Lucas,
it seems like you didn't add any changes to this merge request? It's just the two PRs from ´´dev´´ into your branch? Can you double-check?

**Comment by lsawade** [comment]
Did I mess something up...

**Comment by lsawade** [comment]
> Hi Lucas,
> it seems like you didn't add any changes to this merge request? It's just the two PRs from ´´dev´´ into your branch? Can you double-check?

I you look at the diff, I made a bunch of changes that are hiding in merges...

File: src/pyglimer/waveform/download.py [hunk 1]
```diff
@@ -813,7 +817,7 @@ def downloadwav(
     # # Create handler to the log
     if log_fh is None:
         fh = logging.FileHandler(os.path.join('logs', 'download.log'))
-        fh.setLevel(logging.INFO)
+        fh.setLevel(logging.info)
```
Comment (PeterMakus) [inline src/pyglimer/waveform/download.py:820]: this is not correct. ``logging.info`` is a function, whereas ``logging.INFO`` is the actual logging level.

**Review by PeterMakus — CHANGES_REQUESTED** [review]
seems ok to me. Please change the one point that I commented. Also, I am not sure wrt the impact of ``download_chunk_size_in_mb``

**Comment by PeterMakus** [comment]
looks good. Seems like there is an error with lint (So probably some cosmetic thing). We should probably have a separate action for linting and not run it together with unittest, sorry for that.
After the linting is fixed, we can merge

**Comment by PeterMakus** [comment]
@lsawade Can you check to fix the things that make the unittests fail? If I remember correctly it was only a *listing issue*. But otherwise I think this is almost mergeable.

**Comment by lsawade** [comment]
I have made some more mini-updates that I'm going to push and as part of that I'm going to fix the formatting errors.

**Comment by PeterMakus** [comment]
again some linting problems, but after you fixed this, we can merge.

**Pull request merged**